pldoc/_components/tables.md

```diff
@@ -221,28 +221,22 @@ info:           Tables are used for displaying information in tabular format, wh
       <th scope="col" class="sortable sort-current sort-descending">
         First
         <button class="sort action-sort">
-          <span class="icon-fallback icon-fallback-text">
-            <span class="icon icon-sort" aria-hidden="true"></span>
-            <span class="text">Sort<span class="sr">&nbsp;Ascending</span></span>
-          </span>
+            <span class="icon fa fa-sort" aria-hidden="true"></span>
+            <span class="sr-only">Sort<span class="sr">&nbsp;Ascending</span></span>
         </button>
       </th>
       <th scope="col" class="sortable">
         Last
         <button class="sort action-sort">
-          <span class="icon-fallback icon-fallback-text">
-            <span class="icon icon-sort" aria-hidden="true"></span>
-            <span class="text">Sort<span class="sr">&nbsp;Ascending</span></span>
-          </span>
+            <span class="icon fa fa-sort" aria-hidden="true"></span>
+            <span class="sr-only">Sort<span class="sr">&nbsp;Ascending</span></span>
         </button>
       </th>
       <th scope="col" class="sortable">
         Lineage
         <button class="sort action-sort">
-          <span class="icon-fallback icon-fallback-text">
-            <span class="icon icon-sort" aria-hidden="true"></span>
-            <span class="text">Sort<span class="sr">&nbsp;Ascending</span></span>
-          </span>
+            <span class="icon fa fa-sort" aria-hidden="true"></span>
+            <span class="sr-only">Sort<span class="sr">&nbsp;Ascending</span></span>
```

Comment:
We don't need both `sr` and `sr-only`. In fact, I think only `sr-only` works. Just wrap them together like this:

`<span class="sr-only"> Sort ascending</span>`